### PR TITLE
add GLSL_cluster_culling_shader

### PR DESCRIPTION
**Pull request by wycwang0823**

### CODE REVIEW

**Comment by wycwang0823** [comment]
hi Piers,
I have signed the [Contributor License Agreement], Is there anything else I need to do before merge?

**Comment by dgkoch** [comment]
>  I have signed the [Contributor License Agreement], 

The CLA checker doesn't seem to think that you have signed the CLA yet, can you please double check that you've done it?

> Is there anything else I need to do before merge?

The new extension should also be listed in the README.md so that it shows up in the index.

**Comment by dgkoch** [comment]
> The CLA checker doesn't seem to think that you have signed the CLA yet, can you please double check that you've done it?

This could be because the Author that created the commit is different from the one submitting the PR, if they are both the same person, update the author on the original commit, or otherwise have them sign the CLA as well.

**Comment by wycwang** [comment]
Hi Daniels,
I have signed the CLA, could you please help to merge these commits?

File: extensions/huawei/GLSL_HUAWEI_cluster_culling_shader.txt [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+	YuChang Wang , HUAWEI	
+
+
+Contributors	
+	
+ 	YuChang Wang, HUAWEI	
+
+Status	
+
+	Draft
```
Comment (dgkoch) [inline extensions/huawei/GLSL_HUAWEI_cluster_culling_shader.txt:19]: is this still draft?

**Review by dgkoch — APPROVED** [review]

**Pull request merged**